index.html

```diff
@@ -36,29 +41,36 @@
           placeholder="Search..."
         />
         <button class="nav-search--button" type="button">Search</button>
-
         <a href="#">Login</a>
       </div>
     </nav>
 
     <!-- CAROUSEL -->
     <div class="carousel">
-      <figure class="carousel-container">
-        <img
-          src="https://placeimg.com/1000/600/tech"
-          alt="product-image"
-          class="carousel-container-image"
-        />
-      </figure>
-      <a class="prev"></a>
-      <a class="next"></a>
-    </div>
-    <br />
-
-    <div style="text-align: center">
-      <span class="dot"></span>
-      <span class="dot"></span>
-      <span class="dot"></span>
+      <div class="slides">
+        <figure class="carousel-container">
```

Comment:
Seems like a very correct element for this case, but you could also consider using `figcaption` element for the item text instead of a `div`. 😃